### ISSUE TITLE
Found warnings when testing out your TPAR report.

### ISSUE THREAD

**Issue by Joe23232** [darcys22/tpar#6]
Hey Sean,

So I went to the government's website and they have a [TPAR report testing](https://softwaredevelopers.ato.gov.au/portal-bde)

![image](https://user-images.githubusercontent.com/34926497/129291117-4d9365ca-7ce2-4491-a661-8215ac67c2b8.png)

After uploading the TPAR report your website generated, I got some warnings.
```
Summary of Warnings:
--------------------

Reason Code:  WM04                      Total warnings:  2
The field must not contain blanks or non numeric characters.

Detail of Warnings:
--------------------

Reason Record    Record          Reporting         Field Name                    Field Data
Code:  No.       Type            Reference
WM04   6         Payee           Darcy Financial   Fin institution account no    00000000
WM04   6         Payee           Darcy Financial   Financial institution BSB     00000
```
![image](https://user-images.githubusercontent.com/34926497/129291223-21cccb09-bde4-4d5c-9788-fba0e83c42a6.png)
![image](https://user-images.githubusercontent.com/34926497/129291265-bce68b1f-0764-40bc-a78e-e9fd9cec6003.png)

these were the only fields that was missing, and I wanted to make sure that this is nothing I need to be concerned about?

Kind regards,

Joe

**Comment by darcys22** [comment]
Yeah they are warnings relating to bank account details which i don't request from the user. The file transfer tester will create errors if you are unable to lodge and warnings are encouraged to be resolved but won't prevent lodging

**Comment by Joe23232** [comment]
@darcys22 Hey man thanks for your response.

So when I am lodging a TPAR is it required to put in bank details though? Sorry I just want to make sure I get everything right that is all :)

**Comment by darcys22** [comment]
Should be able to safely lodge without those details

**Comment by Joe23232** [comment]
@darcys22 Cool mate thanks =D
